code/_onclick/item_attack.dm

```diff
@@ -25,7 +25,13 @@
 	I.attack(src, user)
 
 /mob/living/proc/attacked_by(obj/item/I, mob/living/user, def_zone)
-	apply_damage(I.force, I.damtype, def_zone)
+	var/damage = I.force
+	if(mind && mind.demoninfo && mind.demoninfo.banetype == BANEWHITECLOTHES) // Sorry for the snowflake check.
+		if(istype(user, /mob/living/carbon/human))
+			var/mob/living/carbon/human/H = user
+			if(istype(H.w_uniform, /obj/item/clothing/under/color/white))
+				damage *= 3
+	apply_damage(damage, I.damtype, def_zone)

```

Comment:
@lordpidey I would give mobs some kind of "attack intercept" datum, which if returns 1 then normal attackby() isn't called. This attack intercept would allow demon humans to react differently to attacks without it being a snowflake addition, it'd be a generic "attack intercept" system.